### DISCUSSION TITLE
Hey I am trying to create a gallery view [May 15]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2115] [Discord Threads]
> 🙋‍♂️ **MaxWell94**

**Comment by o1lab** [comment]
> 🙋‍♂️ **MaxWell94**  
#### Ok when I noticed the gallery got created  
#  
> 🙋‍♂️ **MaxWell94**  
#### *when I reloaded  
#  
> 💁 **o1lab**  
#### could you please provide a snapshot of your table  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/972789975441604658)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
